driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -856,6 +898,10 @@ protected void processLine(String line, int logLevel) {
     return sw.toString();
   }
 
+  private String execute(String command, Object... args) {
+    return execute(this.ccmDir, command, args);
+  }
```

Comment:
Is it not hanging anymore ?
From our prior threads it should look like this:
```suggestion
  private String execute(String command, Object... args) {
    return execute(logger, this.ccmDir, command, args);
  }
```

Reply:
It was not clear to me what was the exact form suggested, but it's not hanging right now.